clustered/server/src/main/java/org/ehcache/clustered/server/offheap/OffHeapChainMap.java

```diff
@@ -214,6 +214,15 @@ public void put(K key, Chain chain) {
     }
   }
 
+  void remove(K key) {
+    heads.writeLock().lock();
+    try {
+      heads.computeIfPresent(key, (k, v) -> null);
+    } finally {
```

Comment:
Calling computeIfPresent since remove is overridden.

Reply:
What do you mean remove is overridden? I was fairly sure the following would work:
```java
public void remove(K key) {
  heads.writeLock().lock();
  try {
    heads.remove(key).close();
  } finally {
    heads.writeLock().unlock();
  }
}
```